### PR TITLE
[NOT TESTED] Backend,Frontend,Tests: rm txDetail from SignedTx 

### PR DESCRIPTION
**Pull request by aarani**

### CODE REVIEW

**Comment by knocte** [comment]
CI still red (but only one lane left)

File: src/GWallet.Backend/Ether/EtherServer.fs [hunk 1]
```diff
@@ -404,15 +405,8 @@ module Server =
                     return raise <| FSharpUtil.ReRaise ex
             }
 
-        let connectionTimeout =
-            match currency with
-            | Currency.ETC when etcEcosystemIsMomentarilyCentralized ->
-                Config.DEFAULT_NETWORK_TIMEOUT + Config.DEFAULT_NETWORK_TIMEOUT
-            | _ ->
-                Config.DEFAULT_NETWORK_TIMEOUT
-
```
Comment (knocte) [inline src/GWallet.Backend/Ether/EtherServer.fs:413]: @aarani I dont see this explained in the commit msg

**Comment by knocte** [comment]
Unfortunately this PR needs to be rebased. BTW, @aarani re-re-ping on the timeout-config removal.

**Comment by knocte** [comment]
Weird that I'm seeing this CI error very often lately:

```
Secret source: None
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v1' (SHA:50fbc622fc4ef5163becd7fab6573eac35f8462e)
Download action repository 'actions/setup-dotnet@v1.7.2' (SHA:51f68377c181a79065c61bd492bd49be4575c439)
Error: An action could not be found at the URI 'https://api.github.com/repos/actions/setup-dotnet/zipball/51f68377c181a79065c61bd492bd49be4575c439' (A481:7EF9:[33](https://github.com/nblockchain/geewallet/actions/runs/6480938090/job/17599192799#step:1:37)7E67:4C8CD7:65267EF7)
```

Can you please:
- Re-run the failed CI job that has the error above. I think it can become green.
- Create a PR that upgrades our checkout and setup-dotnet actions to latest.

**Comment by aarani** [comment]
Superseded by #238

**Pull request closed without merging**